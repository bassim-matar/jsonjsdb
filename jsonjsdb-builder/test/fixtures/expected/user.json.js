jsonjs.data['user'] = [
  ['id', 'name', 'score', 'tag_id', 'folder_id', 'ok2'],
  [1, 'alice 1', 33, 'tag_1', null, null],
  [2, 'bob 1', 92, 'tag_3', null, 2],
  [3, 'alice 2', 151, 'tag_1', null, null],
  [4, 'bob 2', 210, 'tag_1', 222, null],
  [5, 'alice 3', 269, 'tag_1', null, null],
  [6, 'bob 32', 333, 'tag_3', 444, null],
]
