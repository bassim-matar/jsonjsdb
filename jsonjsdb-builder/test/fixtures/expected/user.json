[
  {
    "id": 1,
    "name": "alice 1",
    "score": 33,
    "tag_id": "tag_1",
    "folder_id": null,
    "ok2": null
  },
  {
    "id": 2,
    "name": "bob 1",
    "score": 92,
    "tag_id": "tag_3",
    "folder_id": null,
    "ok2": 2
  },
  {
    "id": 3,
    "name": "alice 2",
    "score": 151,
    "tag_id": "tag_1",
    "folder_id": null,
    "ok2": null
  },
  {
    "id": 4,
    "name": "bob 2",
    "score": 210,
    "tag_id": "tag_1",
    "folder_id": 222,
    "ok2": null
  },
  {
    "id": 5,
    "name": "alice 3",
    "score": 269,
    "tag_id": "tag_1",
    "folder_id": null,
    "ok2": null
  },
  {
    "id": 6,
    "name": "bob 32",
    "score": 333,
    "tag_id": "tag_3",
    "folder_id": 444,
    "ok2": null
  }
]
